### ISSUE TITLE
Crossword page

### ISSUE THREAD

**Issue by FayCross** [thexerteproject/xerteonlinetoolkits#1015]
A couple of bugs with the new page:

- [x] Console shows it's trying to use various pngs which aren't part of develop - they seem to be number images e.g. 1.png
- [x] When you try to create a crossword if often says it can't use some of the words (I understand not all words may fit together to make a grid) but these often change when you refresh. So sometimes it works and sometimes it doesn't. I guess it only makes one attempt to make a grid - if multiple attempts were made and it chose the one that uses the most amount of words this would be better rather than just failing if it can't use all the words

The following bugs are copied from Trello & are from Takeshi's testing:
- [x] Tab goes to next word but Shift+Tab does not go to previous.
- [ ] Intermittent: If word is completely blank then can type whole word straight in, but if partially filled, does not hop to next box after typing letter. Using arrow key to go to next box doesn't go to where you would expect.
- [ ] Intermittent?: moving around grid with arrow keys, highlighting jumps to unrelated area
- [x] Hint editor has full toolbar but any formatting, images etc. do not appear on page.
- [x] Generated grid is sometimes bigger than visible area and ends up with a vertical scrollbar. Could clues take up smaller part of page? (give width option of small/medium/large in editor?)
- [x] Check button very wide
- [x] No instructions area
- [x] NB. Not case-sensitive but capitalisation will appear as typed in Word field if click "Show answers"

Wishlist:
- [x] Feedback text after check answers
- [x] Picture clues would be nice, maybe appear on page when clue is selected

**Comment by ingedonke** [comment]
@FayCross Thx for testing. Can you share your example with me in https://xot310beta.dlearning.nl/ so I can try to reproduce this?

**Comment by FayCross** [comment]
@ingedonke I've just shared it with you. The first page is never created successfully - normally it just says it's because the word 'bean' can't be included but if you refresh it a few times you sometimes get a longer list of words that won't fit. The second page where I've removed the word bean seems to always work. On page 1 I think it would be better to just load the words it can in a crossword rather than just show a blank screen after the error message.
